### ISSUE TITLE
Crayon code not showing on facebook instant articles

### ISSUE THREAD

**Issue by dingo-d** [aramk/crayon-syntax-highlighter#367]
Hi!

On my site [madebydenis.com](http://madebydenis.com/) I have articles with code in them. On mobile, desktop etc. all shows fine, but when I try to open the article from pages manager, or fb app, the code doesn't show.

Instead I see stuff like [crayon-{some long combination of numbers and letters}]. For iinstant articles I am using [Facebook Instant Articles & Google AMP Pages by PageFrog](https://wordpress.org/plugins/pagefrog/) plugin.

Can you tell me if this is Crayon issue, or this plugins issue that they are not formatting it in a proper way?

Thanks

**Comment by shehzadkhawar** [comment]
Did you solve the issue?

**Comment by dingo-d** [comment]
Nope, I'll have to see how to solve this issue when I'll be redesigning my site...
